packages/@aws-cdk/aws-lambda/test/layers.test.ts

```diff
@@ -79,12 +78,12 @@ describe('layers', () => {
     });
 
     // THEN
-    expect(canonicalizeTemplate(SynthUtils.toCloudFormation(stack))).toHaveResource('AWS::Lambda::LayerVersion', {
+    Template.fromStack(stack).hasResource('AWS::Lambda::LayerVersion', {
       Metadata: {
-        'aws:asset:path': 'asset.Asset1Hash',
+        'aws:asset:path': 'asset.8811a2632ac5564a08fd269e159298f7e497f259578b0dc5e927a1f48ab24d34',
```

Comment:
Wait what? How does this change?

Reply:
It's because we don't have `canonicalizeTemplate()` in the `assertions` module.

Reply:
This will change when the code in the current package changes, so it's not going to be a cross-module test breakage. 
Feels like an ok change to take.

I can also change this to a validation that doesn't depend on the asset hash by extracting the resource from the template.